packages/checkbox/src/icons.tsx

```diff
@@ -0,0 +1,31 @@
+import { shadeBlack, shadeWhite } from '@littlespoon/theme/lib/colors/token'
+
+export const CheckedBoxIcon = () => (
```

Comment:
Actually, my bad, I think you are right in terms of performance since this it's converted into an object at build-time, so it shouldn't affect runtime. It's still more flexible though.